node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -456,13 +475,19 @@ struct State {
 	/// For each relay parent and para id we may be connected to a number
 	/// of collators each of those may have advertised a different collation.
 	/// So we group such cases here.
-	requested_collations: HashMap<(Hash, ParaId, PeerId), PerRequest>,
+	requested_collations: HashMap<PendingCollation, PerRequest>,
 
 	/// Metrics.
 	metrics: Metrics,
 
 	/// Span per relay parent.
 	span_per_relay_parent: HashMap<Hash, PerLeafSpan>,
+
+	/// Keep track of all fetch collation requests
+	collations: FuturesUnordered<BoxFuture<'static, PendingCollationFetch>>,
+
+	/// Keep track of all seconded Collations
+	seconded: HashMap<Hash, CollationEvent>,
```

Comment:
are these really seconded collations or collations being seconded (can be invalid, right?)?